/**:
  zed_node:
    ros__parameters:
      general:
        camera_model: "zed2i"
        serial_number: 38888324
        optional_opencv_calibration_file: "/home/ros/zed_cals/ffc_calibration3.yaml"
      svo:
        # svo.svo_path: "" # Path must be absolute
      pos_tracking:
        publish_tf: false
        publish_map_tf: false
      sensors:
        publish_imu_tf: false
        sensors_image_sync: false
      debug:
        debug_common: false
        debug_point_cloud: false
